apps/common/tests/python/mediawords/languages/test_lt.py

```diff
@@ -17,7 +17,7 @@ def test_sample_sentence(self):
 
     def test_stop_words_map(self):
         stop_words = self.__tokenizer.stop_words_map()
-        assert "buvo" in stop_words
```

Comment:
`buvo` literally means `was` in Lithuanian :) Weird that the "ISO" stopwords list didn't include a term like that.